addon/decorators/validation/after-date.js

```diff
@@ -8,11 +8,11 @@ export function AfterDate(property, validationOptions) {
       name: "afterDate",
       target: object.constructor,
       propertyName: propertyName,
-      constraints: [property],
+      constraints: [property, validationOptions ?? 'second'],
```

Comment:
Looking into the class validator [docs](https://github.com/typestack/class-validator#custom-validation-decorators), it  
appears that `validationOptions` is expected to be an object.

I'm concerned that passing a string may clobbering something internally.

Perhaps change it to have it invoked in this form `@BeforeDate('endDate', {granularity: 'day'})`.  That would also leave us the option to pass other arguments, like the `message` attribute that we've discussed.

Reply:
Great catch, fixed! Also enjoy my use of both optional chaining and null coalescing at the same time!!

Reply:
sweet! looks good.